styles.css

```diff
@@ -0,0 +1,72 @@
+html, body {
+  height: 100%; /* Ensure the body takes full height */
+  margin: 0; /* Remove default margin */
```

Comment:
What is the point of the comments here? Can you explain what you are saying with them?

Reply:
I am just making the children take up the full space and also to have a consistent layout control.